google-startup-scripts/usr/lib/systemd/system/google-shutdown-scripts.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Google Compute Engine user shutdown scripts.
+After=local-fs.target network-online.target network.target
+After=google.service rsyslog.service
```

Comment:
I don't think you want google.service here as this is for shutdown and google.service will have already run long ago.